gputop/remote/Makefile.am

```diff
@@ -48,7 +48,7 @@ gputop-nodejs-toolkit.js
 
 nodemodulesdir = $(libdir)/nodejs/gputop
 nobase_dist_nodemodules_DATA := \
-   $(shell find ./node_modules/{protobufjs,bytebuffer,long,uuid,ws,http,string_decoder,pkill,ultron,options,jsdom,jquery} -type f -name '*.js' -o -name '*.json' -o -name '*.node')
+    $(shell cp -r ./node_modules $(libdir)/nodejs/gputop)
```

Comment:
if possible it could be good to keep what we copy fairly explicit so we don't loose track of what we depend on and to minimize what we install. I don't see a new require being added (except for pkill which we were copying before) so wonder if this was just something that helped while developing/debugging and maybe isn't needed now?